Tokenization_Samples/README.md

```diff
@@ -1,55 +1,81 @@
-                                                VTS samples
- 
- vts_tokenize
--------------------
+# VTS samples
+
+## tokenize
+
 Tokenize a given data or plaintext string using the tokengroup and tokentemplate.
 
-tokengroup : Defines a group name space in the configuration database. The VTS GUI Administrator must create token groups that match the tokengroup parameter names used in the application code.
+**tokengroup** : `Defines a group name space in the configuration database. The VTS GUI Administrator must create token groups that match the tokengroup parameter names used in the application code.`
```

Comment:
Maybe rename "The VTS GUI Administrator" to "Tokenization Server Administrator"  to avoid the VTS keyword.